<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayersTab</class>
 <widget class="QWidget" name="PlayersTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="toolTip">
        <string>Add player</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/add-player-22.png</normaloff>:/icons/icons/add-player-22.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="toolTip">
        <string>Edit player</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/edit-22.png</normaloff>:/icons/icons/edit-22.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="toolTip">
        <string>Delete players</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/icons/delete-player-22.png</normaloff>:/icons/icons/delete-player-22.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FiltrEdit" name="editFiltr">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Filter...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treePlayers">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowPlayersDetails">
       <property name="text">
        <string>Show player's details</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FiltrEdit</class>
   <extends>QLineEdit</extends>
   <header>widget/filtredit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
